File: 242.valid-anagram.php
<?php
/*
 * @lc app=leetcode id=242 lang=php
 * @lcpr version=21909
 *
 * [242] Valid Anagram
 */

// @lc code=start
class Solution {

    /**
     * @param String $s
     * @param String $t
     * @return Boolean
     */
    function isAnagram($s, $t) {
        if(strlen($s) != strlen($t))
            return false;
        $s_count = [];
        $t_count = [];
        for ($i = 0; $i < strlen($s); $i++) {
            $s_count[$s[$i]]++;
        }
        for ($i = 0; $i < strlen($t); $i++) {
            $t_count[$t[$i]]++;
        }
        return $s_count == $t_count;
    }
}
// @lc code=end



/*
// @lcpr case=start
// "anagram"\n"nagaram"\n
// @lcpr case=end

// @lcpr case=start
// "rat"\n"car"\n
// @lcpr case=end

 */

File: 242.valid-anagram.py
#
# @lc app=leetcode id=242 lang=python3
# @lcpr version=21909
#
# [242] Valid Anagram
#

# @lc code=start
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        from collections import Counter
        s_count = Counter(s)
        t_count = Counter(t)
        return s_count == t_count
# @lc code=end



#
# @lcpr case=start
# "anagram"\n"nagaram"\n
# @lcpr case=end

# @lcpr case=start
# "rat"\n"car"\n
# @lcpr case=end

#

